vibes/vibed-ui/src/platform/linux/LinuxPlatform.cpp

```
@@ -0,0 +1,165 @@
+            gc = 0;
+        }
+        if (window != 0) {
+            XDestroyWindow(display, window);
+            window = 0;
+        }
+        XCloseDisplay(display);
+        display = nullptr;
+    }
+}
+
+bool LinuxPlatform::createWindow(int w, int h, const char* title) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)